launcher/server/src/escrow/index.ts

```diff
@@ -0,0 +1,94 @@
+import { escrow as escrowSchema } from '../schemas/escrow';
+import { getS3 } from '../plugins/s3';
+import { getEscrow } from '../plugins/escrow';
+import { getWeb3Client } from '../plugins/web3';
+import { getStripeClient } from '../plugins/stripe';
+import { getCurrency } from '../plugins/currency';
+import {
+  ChainId,
+  ESCROW_NETWORKS,
+  IEscrowNetwork,
+} from '../constants/networks';
+
+export const createEscrow = async (
```

Comment:
Why do we need this?

Reply:
Hi @ArnobChowdhury I would suggest you here not to update the launcher. I think just have the RabbitMQ in launcher servicer and call the endpoint from there only!!
Otherwise it would be very difficult to merge any updates to launcher server from human team!!